otx/algorithms/detection/adapters/mmdet/task.py

```diff
@@ -399,7 +402,13 @@ def hook(module, inp, outp):
             if raw_model.__class__.__name__ == "NNCFNetwork":
                 raw_model = raw_model.get_nncf_wrapped_model()
             if isinstance(raw_model, TwoStageDetector):
-                saliency_hook = ActivationMapHook(feature_model)
+                test_pipeline = cfg.data.test.pipeline
+                width, height = None, None
+                for pipeline in test_pipeline:
+                    width, height = pipeline.get("img_scale", (None, None))
```

Comment:
Is there a better way to get input image size?

Reply:
Yes, mm_dataset is the actual dataset for inference. You can use it to find input tensor's size